ArduCopter/mode_autotune.cpp

```diff
@@ -417,6 +417,61 @@ void Copter::ModeAutoTune::run()
     }
 }
 
+#if AUTOTUNE_ENABLED == ENABLED
```

Comment:
This file already has `#if AUTOTUNE_ENABLED == ENABLED` on the top, so this line must be removed.